src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js

```diff
@@ -214,7 +216,7 @@ const GetVetCenterLocations = `
   query GetVetCenterLocations($onlyPublishedContent: Boolean!) {
     nodeQuery(limit: 1000, filter: {
       conditions: [
-        { field: "status", value: ["1"], enabled: $onlyPublishedContent },
+        { field: "status", value: [${contentStatus}], enabled: $onlyPublishedContent },
```

Comment:
I think it would be better to change the value of `enabled` flag rather than `value`. That way we'll get both published and unpublished content.